quickstarts/microsoft.maps/maps-use-sas/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,97 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
```

Comment:
does the vault & msi need to be colocated?  if so, then need to output the value and pass in to the other deployment.

Reply:
Good best practice, I outputted location from prereq and passed to main template.